pages/test_analytics/test_ownership.md

```diff
@@ -0,0 +1,111 @@
+# Test ownership
+
+Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign test ownership to [Teams](/docs/team-management/permissions).
+
```

Comment:
I reckon we should do a little marketing type explanation around the problem and the solution, something like... 

```suggestion

Test ownership is critical to adopting a healthy testing culture at your organization. By defining test owners, your teams will become accountable for maintaining a fast and reliable test suite, ensuring confidence when you deploy your code.

```

This was a first pass crack at it... @mbelton-buildkite might have good words, but he's also likely slammed so we can always iterate.

Reply:
Looks like this was moved to the top. All good.